manifests/plugin/remote_execution/ssh_user/params.pp

```diff
@@ -0,0 +1,31 @@
+# Remote Execution SSH user default parameters
+class foreman_proxy::plugin::remote_execution::ssh_user::params {
+
+  if defined('$::remote_execution_create_user') {
```

Comment:
Would this be equivalent and more readable: `$manage_user = str2bool(pick(fact('remote_execution_create_user'), 'false'))`

Reply:
Are parameters that come from an ENC added to the $facts hash though?

Reply:
I thought they were.

Reply:
```puppet
notify { 'Top scope var':
  message => $::puppetmaster,
}
notify { 'Fact hash':
  message => $facts['puppetmaster'],
}
```
```
Notice: puppet5.example.com
Notice: /Stage[main]/Base/Notify[Top scope var]/message: defined 'message' as 'puppet5.example.com'
Notice: Fact hash
Notice: /Stage[main]/Base/Notify[Fact hash]/message: defined 'message' as 'Fact hash'
```
The `message` parameter in the second notify is `undef` so ends up defaulting to the resource title.

Reply:
That is interesting. I always assumed they were treated as facts because they all end up in the top scope. Perhaps [getvar](https://github.com/puppetlabs/puppetlabs-stdlib#getvar) works better? Sadly the stdlib version doesn't accept a default (like the puppet 6+ built in version).

Reply:
Updated to use `getvar`